training/controllers/api.py

```diff
@@ -139,44 +137,35 @@ def sign_in():
                     }
                     return jsonify(body), 200
                 else:
-                    return "Bad password", 450
+                    return make_response(jsonify({'Error': 'Bad password!'}), 401)
 
 
 @bp.route('/verify/email/<string:username>')
 def verify_email(username):
     our_user = User.query.filter_by(id=username).first()
 
     if our_user is None:
-        abort(404)
-
-    if our_user is None:
-        return "User not exist", 450
+        return make_response(jsonify({'Error': 'User does not exist!'}), 404)
     else:
         if our_user.mail_validation:
-            return "The email {} was already validated.".format(our_user.username), 205
+            return jsonify({"Detail": "The email {} was already validated.".format(our_user.username)}), 202
         else:
             our_user.mail_validation = True
             db.session.commit()
-            return "Email from user {} was successfully validated.".format(our_user.username), 200
+            return jsonify({"Detail": "Email from user {} was successfully validated.".format(our_user.username)}), 200
 
 
 @bp.route('/users/data/<string:username>')
 def return_user_data(username):
     our_user = User.query.filter_by(id=username).first()
 
     if our_user is None:
-        abort(404)
+        return make_response(jsonify({'Error': 'User does not exist!'}), 404)
```

Comment:
Why do you need to call `make_response`? It looks that sometimes you use it and some times not